### PR TITLE
Add webtorrent-cli

### PR DESCRIPTION
**Pull request by capaldo**
#### New App Submission

**App name:** webtorrent-cli

**Repo or homepage link:** https://github.com/feross/webtorrent-cli

**Description:** Feature packed torrent client from your terminal.

**Why you think it's awesome:** It allows for streaming of the torrents being downloaded via local server.

### CODE REVIEW

**Comment by jneidel** [comment]
@capaldo Please remove the entries from the idea list on a different branch, that way even if the PR adding the app does not get merged the ideas.txt file gets smaller:+1:

File: readme.md [hunk 1]
```diff
@@ -260,6 +260,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [s](https://github.com/zquestz/s) - Open a web search in your terminal.
 - [google-font-installer](https://github.com/lordgiotto/google-font-installer) - Download and install Google Web Fonts on your local machine.
 - [aria2](https://github.com/tatsuhiro-t/aria2) - HTTP, FTP, SFTP, BitTorrent and Metalink download utility.
+- [webtorrent-cli](https://github.com/feross/webtorrent-cli) – Feature packed torrent client from your terminal.
```
Comment (jneidel) [inline readme.md]: I would sort it with the other torrent programms under [files](https://github.com/agarrharr/awesome-cli-apps#files)

Comment (jneidel) [reply readme.md]: Also the [contributing guidelines](https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md) state:
**Additions should be added to the bottom of the relevant category.**

Comment (jneidel) [reply readme.md]: Maybe also update the description:
`Feature packed torrent client from your terminal.` is all marketing bs, without real content. I like to strip the description down to be simple and informative. Also this does not really describe what makes webtorrent unique. I'd rather write the description as:
`Streaming torrent client.`
which is pretty close to their repo description and describes the app much better than some marketing platitude.

Comment (jneidel) [reply readme.md]: Don't take this as a dig at you, it just makes more sense [than a big cleanup afterwards](https://github.com/agarrharr/awesome-cli-apps/commit/20f4498615a283c0ceadcbe853a85ab79a15cd31) :smile:

Comment (capaldo) [reply readme.md]: Not at all! I agree with your input. I'll change it now.

Comment (capaldo) [reply readme.md]: Changes have been made

**Pull request merged**